terraform/groups/ecs-service/main.tf

```diff
@@ -76,7 +77,7 @@ module "ecs-service" {
 }
 
 module "secrets" {
-  source = "git@github.com:companieshouse/terraform-modules//aws/ecs/secrets?ref=1.0.228"
+  source = "git@github.com:companieshouse/terraform-modules//aws/ecs/secrets?ref=1.0.293"
 
   name_prefix = "${local.service_name}-${var.environment}"
   environment = var.environment
```

Comment:
Can you add read_only_root_filesystem = true into this file as well please? 
Cheers!